lib/tls.js

```diff
@@ -13,6 +13,7 @@ var constants = require('constants');
 var Timer = process.binding('timer_wrap').Timer;
 
 var DEFAULT_CIPHERS = 'ECDHE-RSA-AES128-SHA256:AES128-GCM-SHA256:' + // TLS 1.2
+                      'PSK-AES256-CBC-SHA:PSK-3DES-EDE-CBC-SHA:PSK-AES128-CBC-SHA:PSK-RC4-SHA:' + //TODO:SPC PSK adding to default ciphers
```

Comment:
Per https://en.wikipedia.org/wiki/RC4 I believe that RC4 is no longer considered secure so we probably shouldn't offer it as a default cipher option. And yes, I see that it is being supported in TLS 1.0 but that is for bad historical reasons and should probably also be removed.